AGENTS.md

```
@@ -0,0 +1,151 @@
+| Library                | React@19, Motion                              |
+| JS bundler             | Rollup (SWC)                                  |
+| CSS processing         | Emotion                                       |
+| Linting                | ESLint@9                                      |
+| Formatting             | Prettier                                      |
+| Testing                | Jest, Storybook, Chromatic(visual regression) |
+
+## Project structure
+
+nex-ui
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)